internals-js/src/federation.ts

```diff
@@ -311,6 +311,44 @@ function isFieldSatisfyingInterface(field: FieldDefinition<ObjectType | Interfac
   return field.parent.interfaces().some(itf => itf.field(field.name));
 }
 
+function validateInterfaceRuntimeImplementationFieldsTypes(
```

Comment:
Maybe just add some comments around exactly what this function does.

Reply:
Added, thanks.